app/helpers/dashboard_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 module DashboardHelper
+  def max_shown_vms
+    10
```

Comment:
We already have the concept of `AppSettings`. Maybe this could be applied here as well?

Reply:
Changed it to use AppSetting

Reply:
:+1: